openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_load.json

```diff
@@ -91,6 +91,23 @@
                     "key": "yetiRig"
                 }
             ]
+        },
+        {
+            "type": "dict",
+            "collapsible": true,
+            "key": "reference_loader",
+            "label": "Reference Loader",
+            "children": [
+                {
+                    "type": "text",
+                    "label": "Naming",
+                    "key": "naming"
+                },
+                {
+                    "type": "label",
+                    "label": "Here's a link to the doc where you can find explanations about customing the naming of referenced assets: https://openpype.io/docs/admin_hosts_maya#load-plugins"
+                }
+            ]
```

Comment:
Thank you.  Once I lost some time searching the variables I could use for a specific settings, so I decided to put here a direct link to the doc to avoid people losing time :)

Reply:
Still not sure if we should be adding this label like this. As far as I know no other settings do this either so I'm wondering whether it fits the "style" of OpenPype to start doing this now. @mkolar @iLLiCiTiT thoughts?